### PR TITLE
vala-panel-appmenu: Update to 24.02

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Switch to new version scheme
- BAMF now removed. Importer now a standalone library.
- Build xfce4-panel-plugins
- Add monitoring.yml

**Test Plan**

Install on Budgie, XFCE, and MATE

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/v/vala-panel-appmenu/pspec_x86_64.xml [hunk 1]
```diff
@@ -21,19 +21,30 @@
...
             <Path fileType="data">/usr/share/dbus-1/services/com.canonical.AppMenu.Registrar.service</Path>
             <Path fileType="data">/usr/share/defaults/etc/profile.d/appmenu-gtk2-module.sh</Path>
+            <Path fileType="doc">/usr/share/doc/appmenu-gtk-module/AUTHORS</Path>
+            <Path fileType="doc">/usr/share/doc/appmenu-gtk-module/README.md</Path>
+            <Path fileType="doc">/usr/share/doc/appmenu-gtk-module/TODO</Path>
+            <Path fileType="doc">/usr/share/doc/vala-panel-appmenu/README.md</Path>
+            <Path fileType="data">/usr/share/gir-1.0/AppmenuGLibTranslator-24.02.gir</Path>
             <Path fileType="data">/usr/share/glib-2.0/schemas/org.appmenu.gtk-module.gschema.xml</Path>
             <Path fileType="data">/usr/share/glib-2.0/schemas/org.valapanel.appmenu.gschema.xml</Path>
+            <Path fileType="data">/usr/share/licenses/appmenu-gtk-module/LICENSE</Path>
```
Comment (malfisya) [inline packages/v/vala-panel-appmenu/pspec_x86_64.xml:46]: Should I remove these?

File: packages/v/vala-panel-appmenu/pspec_x86_64.xml [hunk 2]
```diff
@@ -21,19 +21,30 @@
...
             <Path fileType="library">/usr/lib64/libappmenu-gtk3-parser.so.0</Path>
-            <Path fileType="library">/usr/lib64/libappmenu-gtk3-parser.so.0.7</Path>
+            <Path fileType="library">/usr/lib64/libappmenu-gtk3-parser.so.24.02</Path>
             <Path fileType="library">/usr/lib64/vala-panel-appmenu/vala-panel/appmenu-registrar</Path>
             <Path fileType="data">/usr/share/dbus-1/services/com.canonical.AppMenu.Registrar.service</Path>
             <Path fileType="data">/usr/share/defaults/etc/profile.d/appmenu-gtk2-module.sh</Path>
+            <Path fileType="doc">/usr/share/doc/appmenu-gtk-module/AUTHORS</Path>
+            <Path fileType="doc">/usr/share/doc/appmenu-gtk-module/README.md</Path>
+            <Path fileType="doc">/usr/share/doc/appmenu-gtk-module/TODO</Path>
+            <Path fileType="doc">/usr/share/doc/vala-panel-appmenu/README.md</Path>
```
Comment (malfisya) [inline packages/v/vala-panel-appmenu/pspec_x86_64.xml:42]: Remove this ?

**Comment by malfisya** [comment]
Also now we have XFCE edition, should we build the XFCE Panel too ?

**Comment by EbonJaeger** [comment]
XFCE panel support would be nice, for anyone who may want it.

File: packages/v/vala-panel-appmenu/pspec_x86_64.xml (hunk 1 above)
Comment (malfisya) [inline packages/v/vala-panel-appmenu/pspec_x86_64.xml:46]: Remove this too ?

File: packages/v/vala-panel-appmenu/pspec_x86_64.xml [hunk 3]
```diff
@@ -21,19 +21,30 @@
...
+            <Path fileType="data">/usr/share/gir-1.0/AppmenuGLibTranslator-24.02.gir</Path>
             <Path fileType="data">/usr/share/glib-2.0/schemas/org.appmenu.gtk-module.gschema.xml</Path>
             <Path fileType="data">/usr/share/glib-2.0/schemas/org.valapanel.appmenu.gschema.xml</Path>
+            <Path fileType="data">/usr/share/licenses/appmenu-gtk-module/LICENSE</Path>
+            <Path fileType="data">/usr/share/licenses/vala-panel-appmenu/LICENSE</Path>
```
Comment (malfisya) [inline packages/v/vala-panel-appmenu/pspec_x86_64.xml:47]: Remove this too ?

**Comment by malfisya** [comment]
I did test this, but need someone that actually use it to see if there is behaiour changes.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Actually, could I ask you to add a `monitoring.yml` file?

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**